### ISSUE TITLE
Circuit's list missing ul element?

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#88]
When reviewing the visual a11y view, I noticed the list of details that describe "The Circuit" are not indented as would be usual in Safari and do not display with bullets in Firefox.

I think the parent `ul` for the list is missing.
http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.9/ohms-law-a11y-view.html

**Comment by terracoda** [comment]
@jessegreenberg or @zepumph, could one of you have a look? I'm not sure where the code is for this. Using the inspector tool, I did verify that I can not see the parent `ul` in the rendered page.

**Comment by jessegreenberg** [comment]
Good catch @terracoda, looks like the parent is a div instead of a ul.

```html
<div tabindex="-1" id="157-218-749-231">
    <li tabindex="-1" id="157-218-749-231-673">Tiny arrows pointing clockwise represent a current of 9.0 milliamps.</li>
    <li tabindex="-1" id="157-218-749-231-233">Exactly 3 batteries show 4.5 volts.</li>
    <li tabindex="-1" id="157-218-749-231-282">A good many dots in resistor represent resistance.</li>
</div>
```

Also missing formatting tags.

**Comment by jessegreenberg** [comment]
Fixed in the above commits, here is the HTML now:
```html
<ul tabindex="-1" id="157-205-736-218">
    <li tabindex="-1" id="157-205-736-218-660"><em>Tiny arrows</em> pointing clockwise represent a current of <em>9.0 milliamps.</em></li>
    <li tabindex="-1" id="157-205-736-218-220"><em>Exactly 3 batteries</em> show <em>4.5 volts.</em></li>
    <li tabindex="-1" id="157-205-736-218-269"><em>A good many dots</em> in resistor represent resistance.</li>
</ul>
```

**Comment by jessegreenberg** [comment]
Closing, thanks @terracoda.
